src/main/java/seedu/address/logic/commands/ShowDescriptionCommand.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelToDo;
+import seedu.address.model.todolist.ToDoListEvent;
+import seedu.address.ui.DescriptionDisplay;
+
+/**
+ * Shows description of a todolist event identified using its displayed index
+ * from the calendar event list in the toDoList.
+ */
+public class ShowDescriptionCommand extends Command {
+
+    public static final String COMMAND_WORD = "show description";
```

Comment:
I feel like something like "show todo" would be a bit simpler, would also make it clear that it is a todo command.

Reply:
ok